patches/tools-perf-core-perf_dashboard_machine_group_mapping.json.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/tools/perf/core/perf_dashboard_machine_group_mapping.json b/tools/perf/core/perf_dashboard_machine_group_mapping.json
+index cd94667a582bb0fb3f1beba888cf356460f80ef6..0f875f716319802d47c3aa4f971f6de6fedab752 100644
+--- a/tools/perf/core/perf_dashboard_machine_group_mapping.json
++++ b/tools/perf/core/perf_dashboard_machine_group_mapping.json
+@@ -1,4 +1,5 @@
+ {
++  "brave.perf": "BravePerf",
```

Comment:
is this really necessary since it's always `BravePerf`? What's the value in adding this?

Reply:
It's necessary to pass the local validation.
The alternatives are 1) patch the validation code (I believe we don't want it) 2) use chromium items.
I believe it's good to have a dedicated group but I can try to switch `chromium.perf`.